Automattic/amp-wp
# Disable CSS tree shaking and CSS limit in Customizer preview

## Summary

* Disable CSS tree shaking in Customizer preview to ensure that CSS for selectively-refreshed partials will be properly styled (e.g. Custom Logo in Twenty Twenty).
* Disable the CSS limit of 75KB in the Customizer preview since tree shaking is disabled.
* Ensure that all styles included in the selective refresh partial will be retained when rendered. This involved gathering up the `style[amp-custom]` generated during rendering and sanitization of the partial to then append that `style` element to rendered partial response. This `style` element gets an `amp-custom-partial` attribute instead of `amp-custom` just to prevent there being confusion.

Fixes #5104

## Testing Instructions

1. Activate Twenty Nineteen as the main theme.
2. Activate Twenty Twenty as the reader theme.
3. Open the AMP Customizer.
4. Select a Custom Logo 
5. Publish the changes
6. Duplicate the tab and reload the page.
7. The Custom Logo should appear styled the same way when selectively-refreshed as fully-refreshed.

Now do the same for removing the logo. The non-logo header should look the same when selectively-refreshed and fully-refreshed.

## Checklist
